pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ObjectFunctions.java

```diff
@@ -18,13 +18,24 @@
  */
 package org.apache.pinot.common.function.scalar;
 
+import java.util.Arrays;
+import java.util.List;
 import javax.annotation.Nullable;
 import org.apache.pinot.spi.annotations.ScalarFunction;
 
 public class ObjectFunctions {
   private ObjectFunctions() {
   }
 
+  private static Object coalesce(List<Object> objects) {
```

Comment:
(minor) Move it to the end following the other coalesce functions

Reply:
Done